### PR TITLE
Terminal website - Initial prototype

### PR DESCRIPTION
**Pull request by The0mikkel**
This pull request adds a version of [LiveTerm by Cveinnt](https://github.com/Cveinnt/LiveTerm) that is *heavily* modified.

The update is *big* as it is the whole project being re-booted.  
This specific code is forked from https://github.com/The0mikkel/themikkel-terminal, as it includes some initial filesystem implementation.  
Filesystem implementation is to be reworked, together with the shell system.

All the items from the previous website are included in this as well, just moved in according to how this Next js system is built.

Included in the update, is a re-worked GitHub flow to build the GitHub page with Next js and Vercel.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,3 @@
-# brunnerne.dk
-Main brunnerne.dk site repo
+# Brunnerne website
+
+Brunnerne website, created used the [LiveTerm by Cveinnt](https://github.com/Cveinnt/LiveTerm) that is *heavily* modified.
```
Comment (Victor4X) [inline README.md]: created using a *heavily* modified version of (Liveterm link here)

File: ascii.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  $$$$$$$                                                                                   
+  $$    $$                                                                                  
+  $$    $$   $$$$$$   $$    $$  $$$$$$$   $$$$$$$    $$$$$$    $$$$$$   $$$$$$$    $$$$$$   
+  $$$$$$$   $$    $$  $$    $$  $$    $$  $$    $$  $$    $$  $$    $$  $$    $$  $$    $$  
+  $$    $$  $$        $$    $$  $$    $$  $$    $$  $$$$$$$$  $$        $$    $$  $$$$$$$$  
+  $$    $$  $$        $$    $$  $$    $$  $$    $$  $$        $$        $$    $$  $$        
+  $$$$$$$   $$         $$$$$$   $$    $$  $$    $$   $$$$$$$  $$        $$    $$   $$$$$$$
+"""
+
+# Replace $ with 0 or 1 randomly
```
Comment (Victor4X) [inline ascii.py:18]: You're right, hiding a message here would be quite neat. Maybe randomly rotate (ROT-n) some secret message?

File: config.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
+{
+  "readmeUrl": "https://raw.githubusercontent.com/cveinnt/cveinnt/master/README.md",
```
Comment (Victor4X) [inline config.json]: Ah, det her er hans README. Prob fine, ved ikke hvor den er brugt på siden.
Ville bare lige gøre dig opmærksom i tilfælde af det var en fejl

File: package.json [hunk 4]
```diff
@@ -0,0 +1,42 @@
+{
+  "name": "liveterm",
```
Comment (Victor4X) [inline package.json]: Det her er fra det tidligere projekt, no?

File: src/utils/bin/api_commands.ts [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+    )
+    .join('\n');
+};
+
+export const quote = async (args: string[]): Promise<string> => {
+  const data = await getQuote();
+  return data.quote;
+};
+
+// export const readme = async (args: string[]): Promise<string> => {
```
Comment (Victor4X) [inline src/utils/bin/api_commands.ts]: Ah, du har fjernet den her

**Review by Victor4X — CHANGES_REQUESTED** [review]
See comments

File: ascii.py (hunk 2 above)
Comment (The0mikkel) [reply ascii.py:18]: Flag added 😎

**Pull request merged**